src/storage/txn/commands/macros.rs

```diff
@@ -183,6 +183,13 @@ macro_rules! gen_lock {
             crate::storage::txn::latch::Lock::new(keys)
         }
     };
+    ($field: ident: enum_match { $( $arm: pat => $expr: expr ), * }) => {
+        fn gen_lock(&self) -> crate::storage::txn::latch::Lock {
+            match &self.$field {
+                $( $arm => crate::storage::txn::latch::Lock::new($expr) ),*
+            }
+        }
+    }
```

Comment:
Where is this rule used?

Reply:
Ah sorry. This is not used now and I forgot to delete it.